cmd/webhook/main.go

```diff
@@ -59,12 +64,12 @@ func main() {
 	// set up signals so we handle the first shutdown signal gracefully
 	stopCh := signals.SetupSignalHandler()
 
-	clusterConfig, err := rest.InClusterConfig()
+	cfg, err := clientcmd.BuildConfigFromFlags(*masterURL, *kubeconfig)
 	if err != nil {
-		logger.Fatal("Failed to get in cluster config", zap.Error(err))
+		logger.Fatal("Error building kubeconfig: %v", err)
```

Comment:
`Fatalf`?

Reply:
Yes, cause BuildConfigFromFlags will try to get config from out of the cluster first, and it will print a warning log if failed. and the it will try to get config from in the cluster, and it will fatalf if failed.
So the Fatalf is ok as it means it can't get config form both in and out of the cluster.

Reply:
Oh, yes.